experimentation/src/test/java/com/automattic/android/experimentation/ExPlatTest.kt

```diff
@@ -15,21 +24,13 @@ import org.mockito.kotlin.times
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.verifyNoInteractions
 import org.mockito.kotlin.whenever
-import org.wordpress.android.fluxc.model.experiments.Assignments
```

Comment:
Suggestion (💡): Consider replacing `runBlockingTest` with `runTest` everywhere.

Reply:
Good suggestion! I applied it in #242 . I'll skip doing this here, to not introduce merge conflicts.

Reply:
Awesome, thanks, will take a look at it there then. 👍